### ISSUE TITLE
Work Doesn't Reset

### ISSUE THREAD

**Issue by sgpthomas** [luizaugustomm/tomato#27]
If you have a timer running past midnight, then the work doesn't reset that day. I don't know exactly how we should deal with this. If you continue working into the next day, it makes sense to keep the work done. However, you do want it to reset after you sleep and resume work the next day.

**Comment by luizaugustomm** [comment]
I think we should let this issue as it is right now. It would cost development time for a case that is rare to happen IMHO.

Moreover, the user has the option to reset work manually if he wants to. :)

**Comment by sgpthomas** [comment]
yeah I agree. just thought that I would file it away in case we want to deal with this at some point in the future
